downstream/titles/release-notes/master.adoc

```diff
@@ -9,17 +13,16 @@ include::{Boilerplate}[]
 
 include::topics/platform-intro.adoc[leveloffset=+1]
 
-include::topics/aap-24.adoc[leveloffset=+1]
+include::topics/aap-25.adoc[leveloffset=+1]
 
-include::topics/controller-440.adoc[leveloffset=+1]
+include::topics/aap-25-deprecated-features.adoc[leveloffset=+1]
 
-include::topics/eda-24.adoc[leveloffset=+1]
+include::topics/aap-25-removed-features.adoc[leveloffset=+1]
 
-include::topics/hub-464.adoc[leveloffset=+1]
+include::topics/aap-25-changed-features.adoc[leveloffset=+1]
 
-include::topics/operator-240.adoc[leveloffset=+1]
+include::topics/aap-25-known-issues.adoc[leveloffset=+1]
 
-include::topics/docs-24.adoc[leveloffset=+1]
+include::topics/aap-25-fixed-issues.adoc[leveloffset=+1]
 
-// Asynchronous release notes - commented out until 2.5 has asynchronous release note updates
-// include::async/async-updates.adoc[leveloffset=+1]
+include::topics/docs-25.adoc[leveloffset+1]
```

Comment:
```suggestion
include::topics/docs-25.adoc[leveloffset=+1]
```
@sayjadha The missing `=` was causing the build to fail.

Reply:
Thank you, @ariordan-redhat!! Resolved the error.